bitnami/contour/templates/envoy/prometheusrule.yaml

```diff
@@ -0,0 +1,27 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.metrics.prometheusRule.enabled .Values.envoy.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ default .Release.Namespace .Values.metrics.prometheusRule.namespace }}
```

Comment:
Could you modify it as follows, please?
`namespace: {{ default .Release.Namespace .Values.metrics.prometheusRule.namespace | quote }}`